### DISCUSSION TITLE
我是不是回不去了

### DISCUSSION THREAD

**Discussion by bxzwq** [share121/inter-knot#5861] [公告]
我在一个自称是旅行者的口中了解到我所处的地方是提瓦特大陆，我问了他几个问题
1、你是怎么找到我的？你的头顶上有蓝色感叹号。
2、我该怎么回到新艾利都？我不知道你说的是什么地方，但你的任务报酬非常高，我一定会把你带回去的。
3、你是谁？我是空，我旁边飘着的这位是应急食品。
他给我带到了一个奇怪的漩涡这，说我可能能从这里会去，但由于这个漩涡会干扰手机，所以我没有照片

**Comment by fengkong-cjb** [comment]
等星穹列车
